cpp/include/cudf/io/parquet.hpp

```diff
@@ -191,6 +204,14 @@ class parquet_reader_options {
    */
   void enable_use_pandas_metadata(bool val) { _use_pandas_metadata = val; }
 
+  /**
+   * @brief Sets to enable/disable conversion of binary to strings per column.
+   *
+   * @param val Vector of boolean values to enable/disable conversion of binary to string columns.
+   * Note default is to convert to string columns.
+   */
+  void set_convert_binary_to_strings(std::vector<bool> val) { _convert_binary_to_strings = val; }
```

Comment:
Maybe we can pass a `host_span` or const reference of the vector to avoid a copy.

Reply:
This comes from the outside world. We can certainly take a reference, but we leave ourselves open for scope issues in a chunked write situation that may be hard to debug. I'm fine with doing this though as long as that is considered.

Reply:
The `host_span` might not be ideal because (I think) the `options` object is intended to own the vector.
Likewise, the `vector const&` would mean that an explicit copy would need to be made anyway, when `_convert_binary_to_strings = val` is done.

I suspect the intention here is to pass by value, and then adopt the copied value.

Reply:
Right, it's an `optional` to own the vector. Thanks for the clarification.

Reply:
Other vector options are set with 'pass by value' +' move'. I think the same should be done here.
```suggestion
  void set_convert_binary_to_strings(std::vector<bool> val) { _convert_binary_to_strings = std::move(val); }
```
Alternatively, we could be using a host_span and copy into a vector, but that's future work that should cover all cuIO APIs.